swan-cern/Chart.yaml

```diff
@@ -10,4 +10,7 @@ description: The chart to deploy SWAN at CERN
 dependencies:
   - name: swan
     version: 0.0.1
-    repository: file://../swan
\ No newline at end of file
+    repository: file://../swan
+  - name: swan-fluentd
```

Comment:
I think there's a better way of doing this. When I checked the Helm documentation on subcharts, I got the impression that was enough to create a new chart inside `charts/`. I think this would remove the need of declaration in dependencies and would keep the folders organized.
Can you check?